PlayCover/Model/AppSettings.swift

```diff
@@ -22,6 +22,8 @@ struct AppSettingsData: Codable {
     var bypass = false
     var discordActivity = DiscordActivity()
     var version = "2.0.0"
+    var playChain = false
```

Comment:
Since two new keys are being added to the settings file, there should be a decoder function similar to how keymapping does it to provide backward compatibility and prevent erasure of settings set before these keys were added.

Reply:
Done!